views/natural_attractions_views.py

```diff
@@ -37,6 +38,90 @@ def get_single_natural_attraction(id):
     data = get_single(sql, id)
     if not data:
         return {}
-    natural_attraction = Natural_Attraction(data['id'], data['name'])
+    natural_attraction = NaturalAttraction(data['id'], data['name'])
 
     return natural_attraction.__dict__
+
+def get_natural_attraction_by_park_id(park_id):
+    with sqlite3.connect("./national_park.sqlite3") as conn:
```

Comment:
You shouldn't need lines 46-48 since you are connecting to the sql database in the get_all_by_param function